src/renderers/webgl-fallback/utils/WebGLState.js

```diff
@@ -841,6 +844,53 @@ class WebGLState {
 
 	}
 
+	/**
+	 * Sets the vertex state by binding the given VAO and element buffer.
+	 *
+	 * @param {WebGLVertexArrayObject} vao - The VAO.
+	 * @param {WebGLBuffer} indexBuffer - The index buffer.
+	 * @return {boolean} Whether a vertex state has been changed or not.
+	 */
+	setVertexState( vao, indexBuffer = null ) {
+
+		const gl = this.gl;
+
+		if ( this.currentVAO !== vao || this.currentIndex !== indexBuffer ) {
+
+			gl.bindVertexArray( vao );
+
+			if ( indexBuffer !== null ) {
+
+				gl.bindBuffer( gl.ELEMENT_ARRAY_BUFFER, indexBuffer );
```

Comment:
FYI: The index is no part of the VAO anymore similar to `WebGLRenderer`. If not doing this, there are scenarios where the index buffer is not correctly bound which results in a "no elements buffer bound" warning in WebGL. If that happens, the affected primitives won't render.